Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/CommonSettings.java

```diff
@@ -152,22 +168,49 @@ private static void setEnvironment(@NotNull String env, Map<String, String> endP
     }
 
     private static boolean isUsingDeprecatedBaseFolder(String baseDir, String deprecated) {
-        return !FileUtil.isNonEmptyFolder(baseDir) && FileUtil.isNonEmptyFolder(deprecated);
+        File baseFile = new File(baseDir);
+        return !baseFile.exists() && FileUtil.isNonEmptyFolder(deprecated);
     }
 
     private static void initBaseDir(@NotNull String basePath) throws IOException {
         File baseDir = new File(basePath);
         if(!baseDir.exists()){
             FileUtils.forceMkdir(baseDir);
         }
-        Files.setAttribute(baseDir.toPath(), "dos:hidden", true);
+        if (Utils.isWindows()) {
+            Files.setAttribute(baseDir.toPath(), "dos:hidden", true);
+        }
+    }
+
+    private static void moveResourcesToBaseFolder(String baseDir, String deprecatedDir) {
+        final File baseFile = new File(baseDir);
+        final File deprecatedFile = new File(deprecatedDir);
+        Arrays.stream(deprecatedFile.listFiles())
+                .filter(CommonSettings::isToolkitResourceFile)
+                .forEach(file -> moveResourceFileToBaseFolder(file, baseFile));
+        cleanDeprecatedFolder(deprecatedFile);
+    }
+
+    private static boolean isToolkitResourceFile(File file){
+        return RESOURCE_FILE_LIST.stream()
+                .anyMatch(resource -> file.isFile() && StringUtils.containsIgnoreCase(file.getName(), resource));
     }
```

Comment:
why containsIgnoreCase? equals or eualsIgnoreCase(depend on  case sensitive)

Reply:
This is to fit log files, tools will write logs to `corelibs.log`, and generate history log file like `corelibs.log.1` and lock file `corelibs.log.lck`, so use contains instead of equals